js/_codemirror.ts

```diff
@@ -171,6 +171,7 @@ export const extensions : { [key: string]: any } = {
     'pascal':        StreamLanguage.define(pascal),
     'perl':          StreamLanguage.define(perl),
     'php':           php,
+    // TODO piet
```

Comment:
https://github.com/dloscutoff/ascii-piet?tab=readme-ov-file#encoding-specification

That's kind of funny 😂